### DISCUSSION TITLE
Welcome to goleak Discussions!

### DISCUSSION THREAD

**Discussion by sywhang** [uber-go/goleak#88] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ashishsinghin** [comment]
Hi @sywhang, Feeling excited to be a part of the community.
